lib/commands/general.js

```diff
@@ -131,17 +132,37 @@ commands.background = async function (seconds) {
   let {appPackage, appActivity} = await this.adb.getFocusedPackageAndActivity();
   await this.adb.goToHome();
   await B.delay(seconds * 1000);
-  return this.adb.startApp({
-    pkg: this.opts.appPackage,
-    activity: this.opts.appActivity,
-    action: this.opts.intentAction,
-    category: this.opts.intentCategory,
-    flags: this.opts.intentFlags,
-    waitPkg: appPackage,
-    waitActivity: appActivity,
-    optionalIntentArguments: this.opts.optionalIntentArguments,
-    stopApp: false
-  });
+
+  let args;
+  if (this.opts.startActivityArgs && this.opts.startActivityArgs[`${appPackage}/${appActivity}`]) {
```

Comment:
`!_.isEmpty(this.opts.startActivityArgs)`

Reply:
I'm not sure what that would gain us. This is just to make sure the variable has something in it before accessing.